README.md

```diff
@@ -24,6 +24,7 @@ You have several options there:
   - wget: `wget -q -O - https://raw.githubusercontent.com/rancher/k3d/master/install.sh | TAG=v1.3.1 bash`
   - curl: `curl -s https://raw.githubusercontent.com/rancher/k3d/master/install.sh | TAG=v1.3.1 bash`
 
+- (MacOS only!) Use [Homebrew](https://brew.sh): `brew install w3d`a
```

Comment:
Should the name be 'k3d' instead of 'w3d'? 

Is the 'a' at the end of line required?

Reply:
I have no idea what happened.  I have noticed that since Catalina that autocorrect has been much more aggressive but that doesn’t explain the “a”.